backend/de.metas.business/src/main/java/de/metas/product/impl/ProductBL.java

```diff
@@ -446,6 +446,26 @@ public String getProductValue(@NonNull final ProductId productId)
 		return product.getValue();
 	}
 
+	@Override
+	@Nullable
+	public String getEAN13Code(@NonNull final ProductId productId)
+	{
+		final I_M_Product product = getById(productId);
+		if (product == null)
+		{
+			return "<" + productId + ">";
+		}
+
+		final String upc = product.getUPC();
+
+		if(upc == null)
+		{
+
+			return null;
+		}
+		return upc.substring(2, 6);
```

Comment:
why returning substring 2, 6?

Reply:
what is this IPC? is an EAN13?

Reply:
The UPC is a string of length 30, described as: 
```
Use this field to enter the bar code for the product in any of the bar code symbologies (Codabar, Code 25, Code 39, Code 93, Code 128, UPC (A), UPC (E), EAN-13, EAN-8, ITF, ITF-14, ISBN, ISSN, JAN-13, JAN-8, POSTNET and FIM, MSI/Plessey, and Pharmacode)
```

I am returning the substring because that's the part of the UPC which fits the barcode in the example given by the client: 

![image](https://github.com/user-attachments/assets/b10d4f82-f96b-40bf-8453-09553e031cc8)


![image](https://github.com/user-attachments/assets/8d9291b1-6a55-4496-b012-51c5854c4bed)